frontend/services/investors/investorsService.ts

```diff
@@ -0,0 +1,50 @@
+import { useMemo } from 'react';
```

Comment:
I added this service because I thought we was going to get the investor account name from here, but we don't have te endpoint for that.